packages/runtime/src/lib/compare.ts

```diff
@@ -20,8 +20,9 @@ export function compareArray (a: unknown, b: unknown): boolean {
 }
 
 // check if index is accessible in an array
-export function checkIndexBound (idx: number, arr: any[]): void {
+export function checkIndexBound (idx: number, arr: any[], line?: number): void {
+  const lineNumber = line ?? 'unknown';
```

Comment:
I am curious regarding the `unknown` line number. When can the line number be unknown?

Reply:
when user is testing from runtime object, let's say user access account from runtime.getAcc() we don't have line here. but when accessing from interpreter a line is definitely passed(similarly for some other functions).

Reply:
Ok, makes sense. Please add a doc string regarding line number `unknown` too. (if user is accessing this function directly through runtime, the line number is unknown)